app/models/user.rb

```diff
@@ -22,7 +22,18 @@ class User < ApplicationRecord
                                     content_type:  /\Aimage\/.*(jpg|jpeg|png)\z/
   validates_attachment_file_name :member_photo, matches: [/png\z/, /jpe?g\z/, /PNG\z/,/JPE?G\z/]
 
-  validates_presence_of :first_name, :last_name, unless: Proc.new {!new_record? && !(first_name_changed? || last_name_changed?)}
+  validates_presence_of :first_name, :last_name, unless: :updating_without_name_changes
```

Comment:
current rails syntax is: `validates :firtst_name, :last_name, presence: true, unless: ...`
http://api.rubyonrails.org/v5.1.3/classes/ActiveModel/Validations/ClassMethods.html#method-i-validates